trigger/process.go

```diff
@@ -289,3 +296,57 @@ projectLoop:
 
 	return versions, catcher.Resolve()
 }
+
+// TriggerDownstreamProjectsForPush triggers downstream projects for a push event from a repo that does not
```

Comment:
Do we intend to make repotracker disabled a requirement on the UI? Is there a reason why they couldn't define it this way with repotracker enabled?

Reply:
That was what I was envisioning, where disabling repotracker and having no config file would indicate that it was an empty project that only exists to trigger downstream versions from its pushes.  But I guess it is overly limiting, and not really necessary to do it that way.

I also for some reason thought it wasn't possible to enable repotracker with an empty config file in the settings but it looks like that's not the case

Reply:
Created! https://jira.mongodb.org/browse/EVG-20985